.github/workflows/repo2cpe-download.yaml

```diff
@@ -23,14 +26,19 @@ jobs:
     - name: Download repository mappings
       run: |
         curl --fail --silent --show-error --max-time 60 --retry 3 --create-dirs \
-          -o 'redhat-repository-mappings/#1' \
+          -o 'repomapping-tmp/#1' \
           'https://access.redhat.com/security/data/metrics/{repository-to-cpe.json,container-name-repos-map.json}'
-        for f in redhat-repository-mappings/*; do
+        for f in repomapping-tmp/*; do
           jq empty "$f"
         done
 
     - name: Upload repository mappings to Google Cloud Storage
       run: |
+        # Zip the files into repomapping.zip
+        zip repomapping.zip repomapping-tmp/*
```

Comment:
Nit: I would prefer to name the file `mappings.zip` or `redhat-repository-mappings.zip` (yes, copied from the dir).